tests/fdb/test_fdb.py

```diff
@@ -208,7 +207,7 @@ def test_fdb(ansible_adhoc, ptfadapter, duthosts, rand_one_dut_hostname, ptfhost
     # Only take interfaces that are in ptf topology
     ptf_ports_available_in_topo = ptfhost.host.options['variable_manager'].extra_vars.get("ifaces_map")
     available_ports_idx = [ idx for idx, name in ptf_ports_available_in_topo.items()
-    if conf_facts['PORT'][port_index_to_name[idx]].get('admin_status', 'down') == 'up' ]
+    if idx in port_index_to_name and conf_facts['PORT'][port_index_to_name[idx]].get('admin_status', 'down') == 'up' ]
```

Comment:
Could you adjust the indention? Indeed, I have been wondering why there is no syntax error here.

Reply:
Thanks. Updated.